README.md

```diff
@@ -9,7 +9,8 @@ Repository for the UI of Screencrash (also known as Skärmkrock)
 
 
 ## Dependencies
-The primary dependency for this repository is [Node.js](https://nodejs.org/) which needs to be installed and accessible.
+The primary dependency for this repository is [Node.js](https://nodejs.org/) 10.16.0 which needs to be installed and accessible.
+To install a specific version, you can use [NVM](https://github.com/nvm-sh/nvm).
```

Comment:
I realized this is only for Mac and Linux. I found [this](https://github.com/coreybutler/nvm-windows) for Windows but cannot vouch for it. Should I add it anyway?

Reply:
You could add a link to https://nodejs.org/dist/v10.16.0/ for Windows users. All distributables can be found there.

Reply:
How easy is it to change the Node version on Windows if we decide we need some new features or a package that depends on a later version? I just know it's a pain on Linux which is why NVM is so useful.

Reply:
Haven't done it but it's probably just installing a new executable and changing the PATH environment variable so it points to the correct path. The installer might even help you fix the PATH variable if it's kind enough. Shouldn't be a problem.